fundamentals-ts.html

```diff
@@ -1652,80 +1651,70 @@ <h1>Contents</h1>
 
       <!-- These rows are in the same order as their feature appears in this document. -->
       <tbody><tr>
-        <td>N4388</td>
-        <td>A Proposal to Add a Const-Propagating Wrapper to the Standard Library
+        <td>Const-propagating wrapper
         </td><td><cxx-ref to="propagate_const"><a title="propagate_const" href="#propagate_const">6.1</a></cxx-ref>
         </td><td><code>propagate_const</code>
         </td><td>201505
         </td><td><code>&lt;experimental/propagate_const&gt;</code>
       </td></tr>
       <tr>
-        <td>P0052R10</td>
-        <td>Generic Scope Guard and RAII Wrapper for the Standard Library
+        <td>Generic scope guard and RAII wrapper
         </td><td><cxx-ref to="scopeguard"><a title="scopeguard" href="#scopeguard">6.2</a></cxx-ref>
         </td><td><code>scope</code>
         </td><td>201902
         </td><td><code>&lt;experimental/scope&gt;</code>
       </td></tr>
       <tr>
-        <td>N3866</td>
         <td>Invocation type traits</td>
         <td><cxx-ref to="meta.trans.other"><a title="meta.trans.other" href="#meta.trans.other">6.3.2</a></cxx-ref></td>
         <td><code>invocation_type</code></td>
         <td>201406</td>
         <td><code>&lt;experimental/type_traits&gt;</code></td>
       </tr>
       <tr>
-        <td>N4502</td>
-        <td>The C++ Detection Idiom</td>
+        <td>Detection metaprograms</td>
```

Comment:
I think this is not the right summary of the feature. Isn't this `void_t` from Walter?

Reply:
It's not only `void_t`, but `is_detected` and `nonesuch` etc.: https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2015/n4502.pdf

Any suggestions for how to call this?

Reply:
See https://cplusplus.github.io/fundamentals-ts/v3.html#meta.detect

Reply:
Ah, ok. Then, the summary does look plausible. (Not that anyone would deeply care.)

Reply:
True, but I did just make those names up on the fly, so the review is very much welcome.